### ISSUE TITLE
[Feat] Use ChatGPT to translate (with context)

### ISSUE THREAD

**Issue by giln** [FlineDev/ReMafoX#104]
## Problem Statement

Allow to give context for better translations.

## Suggested Solution

Allow ChatGPT as a translation option.

## Additional Considerations

Maybe there could be a field with the context to transmit to Chatgpt before translating: for exemple we could describe the application, it's purpose, domain etc so the translation are more accurate.

**Comment by Jeehut** [comment]
@giln Thank you for requesting this feature! It seems to be an interesting idea and I had thought of it already. I'm just not sure how well it will work, for example it seemed to be offline quite often from my experience, but that's could have changed.

Just to set expectations right: If I understand you correctly, you want to have the option to provide your personal ChatGPT API token in RemafoX and therefore use your own account to translate through ChatGPT, right?

**Comment by giln** [comment]
Yes, just like other translation services, with a field where I can provide my own API key.
From my point of view, chatGPT has become much more reliable lately.

**Comment by Jeehut** [comment]
@giln Thank you for the clarification! I will consider adding that in a future update. 💯 
Don't forget to vote with a 👍 above to help me prioritize.
